apis/vshn/v1/dbaas_vshn_postgresql.go

```diff
@@ -144,6 +144,13 @@ type VSHNPostgreSQLServiceSpec struct {
 
 	// PgBouncerSettings passes additional configuration to the pgBouncer instance.
 	PgBouncerSettings *sgv1.SGPoolingConfigSpecPgBouncerPgbouncerIni `json:"pgBouncerSettings,omitempty"`
+
+	// +kubebuilder:validation:Enum=true;false
+	// +kubebuilder:default=true
+	RepackEnabled bool `json:"repackEnabled,omitempty"`
+	// +kubebuilder:validation:Enum=true;false
```

Comment:
Why do you need a validation enum for a boolean value?

Reply:
fixed